### DISCUSSION TITLE
Filtering messages

### DISCUSSION THREAD

**Discussion by sfederici** [ScratchAddons/ScratchAddons#1150] [Ideas]
Something that I find really annoying (I don't know if this is a Scratch website bug or if there is a reason for this) is when I get notifications when **I** add a project to one of my own studios. Of course I know that I added the project to the studio, so what do I get notified for? I get notified only for some of my studios, not for all of them. For example for the studio https://scratch.mit.edu/studios/4467093/activity/. Would it be possible to add a flag to the messaging addons so that I can filter out notifications of my own additions to my own studios?

**Comment by cobaltt7** [comment]
That’s a topic for Scratch’s Bugs and Glitches form. I have an addon in-the-works for that.

**Reply by sfederici** [reply]
> That’s a topic for Scratch’s Bugs and Glitches form

Yes, I know. Indeed I did post this bug in the Scratch website several times. But it has been always ignored by the Scratch Team until now.

> I have an addon in-the-works for that

This would be really great. Thanks in advance

**Reply by cobaltt7** [reply]
see #1151

**Comment by WorldLanguages** [comment]
Are you talking about the "x studio had activity today" messages (studio activity messages)? Those can't be turned off, and they trigger for any activity, even if all activity was by you.

**Reply by sfederici** [reply]
I know these messages cannot be turned off in the Scratch messaging system. That is why I'm proposing to filter out all the "x studio had activity today" messages in Scratch AddOns messaging when they are related to "x added project y" activities where x is me.

**Reply by WorldLanguages** [reply]
Yeah unfortunately it's not possible to mark a single message as read - and the message count sometimes counts "messages from the scratch team", which aren't actually messages. So it's not really feasible to hide messages and keep the message count working properly.

**Comment by spacerhero132** [comment]
(scratch addons) already has that

**Reply by rgantzos** [reply]
This discussion was opened 3 years ago.

**Comment by sfederici** [comment]
In reality, I still get notification messages when I add project to my studios.
